platform/windows/detect.py

```diff
@@ -288,7 +287,7 @@ def configure_msvc(env, manual_msvc_config):
 
     ## LTO
 
-    if env["use_lto"]:
+    if env["lto"] != "none":
```

Comment:
MSVC build probably should have check for `thin` and return error as well.

Reply:
Good point, done.